src/types/api.ts

```diff
@@ -97,6 +101,35 @@ export interface UserResponse {
   repos: Repository[];
 }
 
+export interface OrgSettings {
+  fileContents: string;
+  lastUpdated: number;
+  editors: string[];
+}
+
+export interface JSONAPIErrorResponse {
+  error: {
+    code: string,
+    message: string,
+  };
+}
+
+export interface JSONAPIDataResponse<T> {
+  data: T;
+}
+
+export type JSONAPIResponse<T> = JSONAPIErrorResponse|JSONAPIDataResponse<T>;
+
+export interface GithubAppInstall {
```

Comment:
This doesn't seem like it belongs in the API. From what I can see, this is used in the API to convert from install id to org name. This would then just need to be a single string `org: string`.